brevbaker/src/main/kotlin/no/nav/brev/brevbaker/BrevbakerPDF.kt

```diff
@@ -0,0 +1,44 @@
+package no.nav.brev.brevbaker
+
+import io.ktor.http.ContentType
+import no.nav.pensjon.brev.PDFRequest
+import no.nav.pensjon.brev.api.model.LetterResponse
+import no.nav.pensjon.brev.api.model.maler.BrevbakerBrevdata
+import no.nav.pensjon.brev.template.Letter
+import no.nav.pensjon.brev.template.render.Letter2Markup
+import no.nav.pensjon.brev.template.render.LetterWithAttachmentsMarkup
+import no.nav.pensjon.brev.template.toCode
+import no.nav.pensjon.brev.template.toScope
+import no.nav.pensjon.brevbaker.api.model.LetterMarkup
+import java.util.Base64
+
+private val base64Decoder = Base64.getDecoder()
+
+internal class BrevbakerPDF(private val pdfByggerService: PDFByggerService) {
```

Comment:
Fra et brukervennlighetsperspektiv, burde vi samlet disse under en paraply på et vis? Sånn at det er intuitivt å finne frem til det du trenger for å ende opp med et rendret brev?

Reply:
Lol. Innser jo nå at det er `Brevbaker`